### PR TITLE
[OBSDEF-7469] Added prop for disabling previous button

### PR DESCRIPTION
**Pull request by payald**
Description:
Added props for disabling previous button as the previous button is being used as Quick-create button in New Object Store.

Screenshots:
disable-previous

### CODE REVIEW

File: src/newWizard/Wizard.stories.tsx [hunk 1]
```diff
@@ -32,16 +32,7 @@ function buildStepData(numberOfSteps: number = 1) {
...
-            <WizardStep
-                id={index}
-                key={index}
-                name={step.name}
-                type={step.type}
-                valid={true}
-                complete={true}
-            ></WizardStep>
-        );
+        return <WizardStep id={index} key={index} name={step.name} type={step.type} valid={true} complete={true} />;
```
Comment (akate1) [inline src/newWizard/Wizard.stories.tsx:35]: can we revert this change ? seems not required in this PR

**Review by akate1 — APPROVED** [review]
LGTM

**Review by psivakumaran — APPROVED** [review]

File: src/newWizard/Wizard.stories.tsx (hunk 1 above)
Comment (payald) [reply src/newWizard/Wizard.stories.tsx:35]: I tried to revert the change but formatter is doing this change, not me. Unable to revert this change.

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM
